WordPress/src/androidTest/java/org/wordpress/android/ui/screenshots/WPScreenshotTest.java

```diff
@@ -137,7 +137,7 @@ private void editBlogPost() {
 
         // Wait for the blog posts to load, then edit the first post
         waitForRecyclerViewToStopReloading();
-        waitForAtLeastOneElementOfTypeToExist(CardView.class);
+        waitForAtLeastOneElementWithIdToExist(R.id.card_view);
```

Comment:
Waiting for `CardView.class` isn't enough here because a placeholder `CardView` is displayed before loading is finished. Using the id fixes this.

Reply:
For some reason, on my machine this one is unreliable either way – in some cases it fails with `CardView.class`, in others, it fails on `R.id.card_view`. 

But on the original machine I wrote it with, it still works fine ☹️

I figure either @loremattei could be the tie breaker, or we could try to find another element to target? 

I'm guessing it's some sort of timing issue with the network, but tough to say.

Reply:
I think I have fixed this now. It turns out the posts list doesn't use the same recycler view as reader so `waitForRecyclerViewToStopReloading()` was having no effect.

I have added a new `waitForSwipeRefreshLayoutToStopReloading()` which waits for the appropriate view.

I'd appreciate if you could test this out and let me know how it goes 😄